src/styles/box.scss

```diff
@@ -0,0 +1,99 @@
+@import 'variables';
+@import 'stars';
+
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid $white-accent;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  transition: transform 0.3s ease;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin-left: 3px;
+  }
+
+  &__title {
+    margin-top: 40px;
+    margin-bottom: 4px;
+    font-weight: 500;
+    font-size: 12px;
+  }
+
+  &__code {
+    color: $secondary;
+    font-size: 10px;
+    line-height: 14px;
+    margin: 0 0 3px;
+  }
+
+  &__group-stars {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  &__price {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin: 11px 0 16px;
+
+    .card__price-title {
+      color: $secondary;
+      margin: 0;
+    }
+
+    .card__price-value {
+      font-size: 16px;
+      font-weight: 700;
+      margin: 0;
+    }
+  }
+
+  &__button-buy {
+    width: 100%;
+    height: 40px;
+    border-radius: 5px;
+    background-color: $blue-accent;
+    border: 1px solid $blue-accent;
+    color: $white-accent;
+    font-size: 14px;
+    line-height: 16px;
+    font-weight: 700;
+    cursor: pointer;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    text-decoration: none;
+    transition: all 0.3s ease-in-out;
+    text-transform: uppercase;
+
+    &:hover {
+      background-color: $white-accent;
+      color: $blue-accent;
+    }
+  }
+
+  &:hover {
+    transform: scale(1.2);
+    z-index: 1;
+  }
+}
+
+
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-size: 12px;
+  line-height: 18px;
+  color: $main-accent;
+  font-family: $main-font;
+}
```

Comment:
better move it to `index.scss` file and change the name of this file to `card.scss`